src/Utilities/CommandLineBuilder.cs

```diff
@@ -503,7 +503,7 @@ public void AppendFileNamesIfNotNull(ITaskItem[] fileItems, string delimiter)
 
         #region Appending switches with quoted parameters
 
-        internal static string FixCommandLineSwitch(string switchName)
+        public static string FixCommandLineSwitch(string switchName)
```

Comment:
If it's not easy that's fine. Just something that should be done at some point or we'll end up having to document it as public. Not a big deal now though.